### ISSUE TITLE
TMUX: Pane scrolling output affect other panes selection

### ISSUE THREAD

**Issue by house-of-vanity** [alacritty/alacritty#2983]
Hi, issue occurs with tmux. When top pane starts scrolling output, every bottom panes starts to move selections too. It works only with top pane. See gif.
![Alacritty](https://btwiusearch.net/tmux.gif)

```
# System
|                  |                               |
|------------------|-------------------------------|
| alacritty        | 0.3.3 (679e67d)
| Operating System | Arch Linux   
| Rust Version     | rustc 1.39.0-nightly (72b2abfd6 2019-08-29)
| Display Server   | X11 
| Window Manager   | KDE Plasma 
| Compositor       | kWin 

```

# Expectation
I checked it with Konsole and it works as expected. See gif.
![Konsole](https://btwiusearch.net/tmux_konsole.gif)

[alacritty.log](https://github.com/jwilm/alacritty/files/3850791/alacritty.log)

[alacritty.events.log](https://github.com/jwilm/alacritty/files/3850794/alacritty.events.log)

**Comment by kchibisov** [comment]
Besides Alacritty scrolling handling. How `konsole` behaves when your selection in an active pane reaches the top of it? I'm about both cases top and bottom.

**Comment by house-of-vanity** [comment]
Hi, in konsole selection just cancelled when reached the top. See gif.
![tmux](https://user-images.githubusercontent.com/4666566/69038736-7b66d700-09fb-11ea-8bed-442944a4bb89.gif)

**Comment by chrisduerr** [comment]
This should be fixed with #3218.

I've tried to model the behavior with scrolling regions based on what I personally think is reasonable, since I did not have Konsole for testing and Termite just clears the selection.

If you're willing to help with testing, please let me know if it behaves like you would expect @house-of-vanity.
